src/VariableQueryEditor.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import { AthenaQuery, AthenaDataSourceOptions } from './types';
-import { QueryEditorProps } from '@grafana/data';
+import { AthenaDataSourceOptions, AthenaQuery } from './types';
+import { CoreApp, QueryEditorProps } from '@grafana/data';
 import { DataSource } from 'datasource';
 import { QueryEditor } from 'QueryEditor';
 
 export function VariableQueryCodeEditor(props: QueryEditorProps<DataSource, AthenaQuery, AthenaDataSourceOptions>) {
-  return <QueryEditor {...props} hideOptions hideRunQueryButtons />;
+  const variableEditorQuery: AthenaQuery = { ...props.datasource.getDefaultQuery(CoreApp.Unknown), refId: 'A' };
```

Comment:
```suggestion
  const variableEditorQuery: AthenaQuery = {
    ...props.datasource.getDefaultQuery(CoreApp.Unknown),
    ...props.query,
    refId: 'A',
  };
```

include `props.query` so `<QueryEditor>` doesn't always receive the default query as a prop.

although longer term `getDefaultQuery` should probably be called higher up the component tree by the component that's managing the query state so that it only gets called when the variable query editor is mounted. i'm not sure where all of that happens for the VariableQueryEditor though. it'd be good if the different query editors didn't have to each call `getDefaultQuery` directly.

Reply:
Yeah currently the variable editor renderer in Grafana core doesn't apply the default query (if provided by the datasource), but it in the future it should.